tests/test_icml_conference.py

```diff
@@ -2205,18 +2205,18 @@ def test_review_stage(self, client, openreview_client, helpers):
                         "description": "Please provide an \"overall score\" for this submission.",
                         "value": {
                             "param": {
-                                "type": "string",
+                                "type": 'integer',
                                 "enum": [
-                                    "10: Award quality: Technically flawless paper with groundbreaking impact, with exceptionally strong evaluation, reproducibility, and resources, and no unaddressed ethical considerations.",
-                                    "9: Very Strong Accept: Technically flawless paper with groundbreaking impact on at least one area of AI/ML and excellent impact on multiple areas of AI/ML, with flawless evaluation, resources, and reproducibility, and no unaddressed ethical considerations.",
-                                    "8: Strong Accept: Technically strong paper, with novel ideas, excellent impact on at least one area, or high-to-excellent impact on multiple areas, with excellent evaluation, resources, and reproducibility, and no unaddressed ethical considerations.",
-                                    "7: Accept: Technically solid paper, with high impact on at least one sub-area, or moderate-to-high impact on more than one areas, with good-to-excellent evaluation, resources, reproducibility, and no unaddressed ethical considerations.",
-                                    "6: Weak Accept: Technically solid, moderate-to-high impact paper, with no major concerns with respect to evaluation, resources, reproducibility, ethical considerations.",
-                                    "5: Borderline accept: Technically solid paper where reasons to accept outweigh reasons to reject, e.g., limited evaluation. Please use sparingly.",
-                                    "4: Borderline reject: Technically solid paper where reasons to reject, e.g., limited evaluation, outweigh reasons to accept, e.g., good evaluation. Please use sparingly.",
-                                    "3: Reject: For instance, a paper with technical flaws, weak evaluation, inadequate reproducibility and incompletely addressed ethical considerations.",
-                                    "2: Strong Reject: For instance, a paper with major technical flaws, and/or poor evaluation, limited impact, poor reproducibility and mostly unaddressed ethical considerations.",
-                                    "1: Very Strong Reject: For instance, a paper with trivial results or unaddressed ethical considerations"
+                                    { 'value': 10, 'description': "10: Award quality: Technically flawless paper with groundbreaking impact, with exceptionally strong evaluation, reproducibility, and resources, and no unaddressed ethical considerations." },
```

Comment:
maybe the description here should remove "10:" and keep only the text
because this description will  be returned in details.presentation

in order to combine 10(rating) and "10: Award..." (presentation) with no duplication we will have to assume the format of description which i think is not ideal

Reply:
Yes. We discussed this with Carlos and I have to remove it. I hope it is clear for the user there is no number. I wonder if the drop-down can build the description joining both values.

Reply:
maybe items require another field to have a template string to indicate how to combine the value and description to be shown in note editor